docs/twig-components/method/haswidgets.md

```diff
@@ -2,5 +2,4 @@
 
 `haswidgets(target)` is a Twig function to ...
 
-
-Source: Bolt
\ No newline at end of file
+Please refer to the [widgets](https://docs.bolt.cm/4.0/templating/widgets) documentation.
```

Comment:
Since this PR is going to be merged into 5.0, links should be updated to link to 5.0.